src/main/java/com/labsynch/labseer/domain/StandardizationDryRunCompound.java

```diff
@@ -65,9 +66,9 @@ public class StandardizationDryRunCompound {
 
 	private boolean asDrawnDisplayChange;
 
-	private int newDuplicateCount;
+	private Integer newDuplicateCount;
 
-	private int existingDuplicateCount;
+	private Integer existingDuplicateCount;
```

Comment:
The reason these were changed from lower case int/boolean to upper case Integer, Boolean is to allow it to default to null.  Even though we aren't doing batch processing in the bbchem implementation, this becomes important in the non-bbchem case so that we can batch process rows and know which rows have been processed and which have not.